### DISCUSSION TITLE
EnBizCard

### DISCUSSION THREAD

**Discussion by elbara93** [vishnuraghavb/EnBizCard#40] [General]
Hi Vishnu, 

I hope you are well.

I want to create an app like EnBizCard, how much can you make it for me since you already have the code and know how to make a few modifications? 

El Bara

**Comment by mwarner485** [comment]
Any roadmap item for being able to save the card profile when creating so that if I need to make major changes, I can just open or import it into the web builder to make changes rather than starting from scratch again?
